docgen/layouts/common/meta.pug

```diff
@@ -24,6 +24,6 @@ html
     meta(content='Boost search on your React apps with Algolia', property='og:description')
     meta(content='React InstantSearch', property='og:site_name')
 
-    title React InstantSearch | #{title}
+    title React InstantSearch | Lightning Fast React and React Native Search
```

Comment:
Got it - yes, let's only update it for the home page for now. I'm OK to use the current tagline, but we might want to adjust this in the future.